README.md

```diff
@@ -1,67 +1,48 @@
-# [![Sinch Logo](https://developers.sinch.com/static/logo-07afe977d6d9dcd21b066d1612978e5c.svg)](https://www.sinch.com)
-
-# .NET SDK
+# Sinch .NET SDK
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/sinch/sinch-sdk-dotnet/blob/main/LICENSE)
 
 [![.NET 6.0](https://img.shields.io/badge/.NET-6.0-blue.svg)](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
 [![.NET 7.0](https://img.shields.io/badge/.NET-7.0-blue.svg)](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
 [![.NET 8.0](https://img.shields.io/badge/.NET-8.0-blue.svg)](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
 
-# Welcome to Sinch's .NET SDK.
-
-Here you'll find documentation to start developing C# code using Sinch services.
+Here you'll find documentation related to the Sinch <language> SDK, including how to install it, initialize it, and start developing <language> code using Sinch services.
 
-To use this SDK you'll need a Sinch account and API keys. Please sign up at [sinch.com](https://sinch.com)
+To use Sinch services, you'll need a Sinch account and access keys. You can sign up for an account and create access keys at [dashboard.sinch.com](https://dashboard.sinch.com).
 
-For more in depth version of the Sinch APIs, please refer to the official developer
-portal - [developers.sinch.com](https://developers.sinch.com/)
+For more information on the Sinch APIs on which this SDK is based, refer to the official [developer documentation portal](developers.sinch.com).
 
->[!WARNING]
->This SDK is currently available as a technical preview. It is being provided for the purpose of
-collecting feedback, and should not be used in production environments.
+> [!WARNING]
+> This SDK is currently available to selected developers for preview use only. It is being provided for the purpose of collecting feedback, and should not be used in production environments.
 
-* [Installation](#installation)
-* [Getting started](#getting-started)
-* [Products](#products)
-* [Handling Exceptions](#handling-exceptions)
-* [Client customization options](#logging-httpclient-and-additional-options)
+- [Installation](#installation)
+- [Getting started](#getting-started)
+  - [Client initialization](#client-initialization)
```

Comment:
Formatting: additional space

Reply:
Intended

Reply:
If there is only one sub-menu, then it's not useful to give it a title. Why did you remove the example that showcases a request?

Reply:
>If there is only one sub-menu, then it's not useful to give it a title.
Can you elaborate?

Reply:
- Menu
  - sub menu 1
  - sub menu 2

Reply:
As you prefer then 😁